src/screens/Notification/Notification.tsx

```
@@ -0,0 +1,105 @@
+      console.error('Error marking notifications as read:', error);
+    }
+  };
+
+  if (loading) return <p>Loading...</p>;
+  if (error) return <p>Error: {error.message}</p>;
+
+  const notifications = data?.user?.notifications || [];
+
+  return (
+    <div className={styles.container}>
+      <h1 className={styles.title}></h1>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)